Kiss.Bff/Extern/ZaakGerichtWerken/Contactmomenten/PostKlantcontactenCustomProxy.cs

```diff
@@ -0,0 +1,194 @@
+            var fullName = $"{firstName} {lastName}".Trim();
+
+            var parsedModel = new JsonObject
+            {
+                ["naam"] = fullName,
+                ["soortActor"] = "medewerker",
+                ["indicatieActief"] = true,
+                ["actoridentificator"] = new JsonObject
+                {
+                    ["objectId"] = email,
+                    ["codeObjecttype"] = "mdw",
+                    ["codeRegister"] = "msei",
+                    ["codeSoortObjectId"] = "email"
+                }
+            };
+
+            parsedModel["medewerkerIdentificatie"] = _getMedewerkerIdentificatie();
+
+            var url = _klantcontactenDestination.TrimEnd('/') + "/klantinteracties/api/v1/actoren";
+
+            var request = new HttpRequestMessage(HttpMethod.Post, url)
+            {
+                Content = JsonContent.Create(parsedModel)
+            };
+
+            _authProvider.ApplyAuthorizationHeader(request.Headers, User);
+
+            var response = await _httpClient.SendAsync(request);
+            response.EnsureSuccessStatusCode();
+
+            var content = await response.Content.ReadAsStringAsync();
+
+            return Ok(content);
+        }
+
+        public async Task<string?> CheckIfActorExists(string email)
+        {
+            var url = $"{_klantcontactenDestination.TrimEnd('/')}/klantinteracties/api/v1/actoren";
+            var queryParams = new Dictionary<string, string>
+            {
+                { "actoridentificatorObjectId", email }
+            };
+
+            var queryString = string.Join("&", queryParams.Where(kv => !string.IsNullOrEmpty(kv.Value)).Select(kv => $"{kv.Key}={kv.Value}"));
+            var requestUrl = $"{url}?{queryString}";
+
+            var request = new HttpRequestMessage(HttpMethod.Get, requestUrl);
+            _authProvider.ApplyAuthorizationHeader(request.Headers, User);
+
+            var response = await _httpClient.SendAsync(request);
+
+            if (response.IsSuccessStatusCode)
+            {
+                var content = await response.Content.ReadAsStringAsync();
+                var jsonResponse = JsonNode.Parse(content);
+
+                var results = jsonResponse?["results"]?.AsArray();
+                if (results != null && results.Count > 0)
+                {
+                    return results[0]?["uuid"]?.ToString();
+                }
+            }
+            return null;
+        }
+
+        public IActionResult LinkActorWithKlantContact(string actorUuid, string klantcontactUuid)
+        {
+            var url = _klantcontactenDestination.TrimEnd('/') + "/klantinteracties/api/v1/actorklantcontacten";
+
+            var payload = new JsonObject
+            {
+                ["actor"] = new JsonObject { ["uuid"] = actorUuid },
+                ["klantcontact"] = new JsonObject { ["uuid"] = klantcontactUuid }
+            };
+
+            return new ProxyResult(() =>
```

Comment:
Zie andere comments. Ik denk niet dat dit een response is die we naar de client willen terugsturen. Je kan wat mij betreft de call doen en dan checken of de response een Ok status code heeft.